src/injectProps.js

```diff
@@ -1,8 +1,8 @@
 function injectProps(target, name, descriptor) {
   const oldFunction = descriptor.value;
 
-  descriptor.value = function propsInjectorFunction(...args) {
-    return oldFunction.bind(this)(...args, this.props);
+  descriptor.value = function propsInjectorFunction() {
```

Comment:
This prevents other people from using the decorator on other methods, like it is describe on the documentation: https://croudcare.github.io/relpers/ 

Maybe we should create another decorator called injectPropsWithArgs?